clean_project/clean_file.py

```diff
@@ -24,7 +26,7 @@ def sanitize(line):
     :param line: string to clean
     :return: new line cleaned
     """
-    return re.sub(ur"[^\w@\d'\s-]+", '', line).encode("utf-8")
+    return re.sub(ur"([^\w\d\s]+|\b\w{1}\b)", ' ', line).encode("utf-8")
```

Comment:
a comment on this regex wouldn't hurt, explaining what it should match and remove exactly